src/components/flowchart/index.js

```diff
@@ -13,10 +11,10 @@ import {
 } from '../../actions';
 import { getLayout, getZoomPosition } from '../../selectors/layout';
 import { getCentralNode, getLinkedNodes } from '../../selectors/linked-nodes';
-import icon from './icon';
+import draw from './draw';
 import './styles/flowchart.css';
 
-const DURATION = 700;
+export const DURATION = 700;
```

Comment:
This might be common in JS land so bear with me.
Why do we have to export the constant instead of making the function take it as a parameter? It feels weird to me the we're importing a function but that only works with the constant in here, and creating that sort of interdependency.

Reply:
As discussed in person: I'm really not too fussed either way, though I narrowly prefer to keep it as a global or make it a property on the class. I've done the latter, but again I don't think it really matters.